src/js/index.js

```diff
@@ -1 +1,44 @@
 import "../styles/main.scss";
+
+const $ = (selector) => document.querySelector(selector)
+
+const emailRegex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+
+$(".js-email-input").addEventListener("keyup", (e) => {
```

Comment:
Lets replace all the jquery here with standard JS. We arent really saving much here by using it. Its just selectors and setting innerHTML its not worth the tradeoff.

Reply:
Personally, I like jQuery. But it should be everywhere (cleaner code) or nowhere (no dependency on jQuery). The general consensus in previous discussions was to avoid jQuery, at least for now, unless necessary.
But specifically for this line: Should it be . (class) or # (id)?

Reply:
Yeah I agree with that. The surface area right now is not worth loading it in.

Sometimes I feel it's used as a comfort thing rather than as a benefit. In this case there is no benefit.

In terms of id or class ... I'm indifferent. If we can use ids on elements that we know will only exist once on the page then that makes me happy but I'm not that fussed

Reply:
This isn't jQuery - see line 3. It just looks like it. It's actually a shorthand for `document.querySelector('.js-email-input').addEventListener...`

Reply:
Yeah I missed that...and I get why - if we ever change to jquery you wont need to refactor but it might also confuse developers around methods that exist upon the return value, not realising its actually returning an `Element`. Not a fan, but it's not a hill i'll die on.